BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoiceViewController.swift

```diff
@@ -380,10 +380,11 @@ extension DigitalInvoiceViewController: DigitalLineItemTableViewCellDelegate {
                 self.invoice?.lineItems[viewModel.index].selectedState = .deselected(reason: nil)
                 return
             }
-            
-            presentReturnReasonActionSheet(for: viewModel.index,
-                                           source: cell.modeSwitch,
-                                           with: returnReasons)
+            if returnAssistantConfiguration.enableReturnReasons {
```

Comment:
This should be moved into the `guard` at line 379. Otherwise the `selectedState` is not set to `.deselected(reason: nil)`.